_data/projects/trivy-resource.yml

```diff
@@ -0,0 +1,11 @@
+name: trivy-resource
+desc: Trivy resource for concourse - Vulnerability scanning of container base image
+site: https://github.com/Comcast/trivy-resource/
+tags:
+  - trivy-resource
+  - trivy
+  - concourse-resource
+  - vulnscan
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/comcast/trivy-resource/labels/up%20for%20grabs
```

Comment:
There's no label listed here which matches this - https://github.com/comcast/trivy-resource/labels

Reply:
My bad. Added it in. Thanks!

Reply:
Fixed all the review comments. Please lemme know if i need to fix anything more. Thanks in advance 😄